lib/ansible/modules/git.py

```diff
@@ -207,6 +207,11 @@
         type: path
         version_added: "2.7"
 
+    git_filter:
+        description:
+            - Can be used for partial clone of a git repo
+        type: str
```

Comment:
Not sure why the tests didn't pick this up but this needs a `version_added: "2.14"` to document when this was added.